Didn't changed anything, just simplified webpack config code

xims opened the pull request:
Didn't changed anything, just 
rearranged and simplified the existing webpack config code, 
removed overwritten code, 
removed duplicated code.

End result is exactly the same but shorter and simpler

catalin-luntraru commented:
Hey @xims , sorry for the late response.
I did try your code changes and the `build-dev` and `build-prod` scripts generate the correct css/js files, but the `start` script doesn't create the css file in memory with webpack-dev-server.
If you provide a fix for that I will gladly merge your pull request.

xims commented:
Hi, Yes, sorry about that. I've been using this setup since and at some stage realised that hot reloader wasn't working for styles. I fixed it in my project and now updated in this repository too.
Please have a look now.
It's less clean than it looked before but still removes some unnecessary code.
